contrib_bots/lib/gcalendar.py

```diff
@@ -0,0 +1,263 @@
+from __future__ import print_function
+from datetime import datetime
+import httplib2
+import logging
+import os
+import re
+
+# Uses Google's Client Library
+#   pip install --upgrade google-api-python-client
+from apiclient import discovery, errors
+from oauth2client import client
+from oauth2client import tools
+from oauth2client.file import Storage
+from oauth2client.tools import argparser
+
+# pip install --upgrade dateparser
+import dateparser
+
+SCOPES = 'https://www.googleapis.com/auth/calendar'
+CLIENT_SECRET_FILE = '.gcal_client_secret.json'
+APPLICATION_NAME = 'Zulip Google Calendar Bot'
+
+credentials = None
```

Comment:
Sorry, this was here because before there were other functions that used these credentials. Fixed.